libtenzir/builtins/connectors/tcp.cpp

```diff
@@ -42,6 +43,32 @@ using tcp_bridge_actor = caf::typed_actor<
   // Write a chunk to the socket.
   auto(atom::write, chunk_ptr chunk)->caf::result<void>>;
 
+struct tcp_metrics {
+  auto emit() -> void {
+    metric_handler.emit({
+      {"port", port},
+      {"handle", handle},
+      {"reads", reads},
+      {"writes", writes},
```

Comment:
Writes being included in this suggests that this is implemented for `save tcp` as well, but it isn't. I think that's misleading.

Reply:
As discussed, let's write a follow-up story for this to get this in quicker.